_data/people.yaml

```diff
@@ -5773,7 +5773,7 @@ sk-sahu:
   pronouns: he/him
   twitter: sangram_ksahu
 sladjana-lukic:
-  affiliation: Https://Www.Adelphi.Edu/Faculty/Profiles/Profile.Php?Pid=0966
+  affiliation: Adelphi University, New York
```

Comment:
Can you remove New York? That could go to the city information

Reply:
> Can you remove New York? That could go to the city information

Done @bebatut